CRM/Utils/System/Backdrop.php

```diff
@@ -642,6 +642,7 @@ public function loadBootStrap($params = [], $loadUser = TRUE, $throwError = TRUE
 
     // CRM-8655: Backdrop wasn't available during bootstrap, so
     // hook_civicrm_config() never executes.
+    // FIXME: This call looks redundant with the earlier call in the same function. Consider removing it.
     CRM_Utils_Hook::config($config);
```

Comment:
Yup.

Reply:
Figuring that one out is kind of its own thing. (Does it actually fire? Sometimes-Never-Always? Was it a legit edge-case? Was it a bad SVN merge? Was it miscommunication? Did it become significant after it was done?)

It doesn't seem likely to affect the essence of this PR, though. A guard based on `$flags['civicrm']` or `$flags['uf']` should be just as effective in both these contingencies:

* If it turns out that the third invocation must be kept, then the important thing is for the third invocation to have different `$flags`. And it would. (It would fire with `[civicrm=>FALSE,uf=>FALSE]`.) 
* If it turns out that the third invocation can be removed, then... it can still be removed. (The PR doesn't make it more or less important than before.)